test/e2e/storage/pd.go

```diff
@@ -405,7 +407,7 @@ var _ = utils.SIGDescribe("Pod Disks", func() {
 					framework.ExpectEqual(numNodes, origNodeCnt, fmt.Sprintf("Requires current node count (%d) to return to original node count (%d)", numNodes, origNodeCnt))
 					output, err = gceCloud.ListInstanceNames(framework.TestContext.CloudConfig.ProjectID, framework.TestContext.CloudConfig.Zone)
 					framework.ExpectNoError(err, fmt.Sprintf("Unable to get list of node instances err=%v output=%s", err, output))
-					framework.ExpectEqual(false, strings.Contains(string(output), string(host0Name)))
+					framework.ExpectEqual(true, strings.Contains(string(output), string(host0Name)))
```

Comment:
Could we just block the code for disruptOp == deleteNode?

Reply:
I would rather not run the test instead of running the test but having it pass when it doesn't actually work

Reply:
I also mean to not run the test, just think this whole part of code is not needed.

Reply:
Yeah I do want to re-enable the test again in the future